t/summary.t

```diff
@@ -0,0 +1,119 @@
+use Test::Most;
```

Comment:
What is the reason to use `Test::Most`? I could only identify `cmp_deeply`, but for that `Test::Deep` would have been enough (or maybe even `is_deeply` without additional test dependencies). And I would keep exact test plans if possible :)

I'm not against using `Test::Most`, but for the sake of consistency, I'd prefer applying this to all (most?) of the other tests too then in a separate PR. Or just importing, what the tests actually needs.